redis/types.go

```diff
@@ -14,24 +14,14 @@ type Config struct {
 	Pass string `env:"REDISPASS" env-description:"Redis user password"`
 }
 
-// repository represent the repository model
-type repository struct {
-	Client redis.Cmdable
+// Store represent the Store model
+type Store struct {
+	client *redis.Client
 }
 
-// Repository represent the repositories
-type Repository interface {
+// KVStore represent the repositories
+type KVStore interface {
```

Comment:
This type should be defined in another package:
pkg/types/stores.go

Reply:
OK. Let me create another pkg for types.